native-assets/fractional-NFT/project/contracts/f-NFT-contract/tests/utils/interface.rs

```diff
@@ -0,0 +1,117 @@
+use crate::utils::setup::F_NFT;
+use fuels::{
+    prelude::{AssetId, CallParameters, TxPolicies, WalletUnlocked},
+    programs::call_utils::TxDependencyExtension,
+    tx::Bytes32,
+    types::{Bits256, Identity},
+};
+
+pub(crate) async fn deposit(
+    contract: &F_NFT<WalletUnlocked>,
+    nft: AssetId,
+    reciever: Identity,
+    vault_sub_id: Bytes32,
+) -> u64 {
```

Comment:
We log events.
In older versions of the SDK we would get the FuelCallResponse to retrieve the logs.
Do we do that differently or is checking logs omitted in this suite atm?

I am about to check the tests themselves so idk yet